rules/macos/defense_evasion_install_root_certificate.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "production"
+updated_date = "2021/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may install a root certificate on a compromised system to avoid warnings when connecting to adversary
+controlled web servers. Root certificates are used in public key cryptography to identify a root certificate authority
+(CA). When a root certificate is installed, the system or application will trust certificates in the root's chain of
+trust that have been signed by the root certificate.
+"""
+false_positives = ["Trusted Root Certificate Install"]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Attempt to Install Root Certificate"
+references = ["https://ss64.com/osx/security-cert.html"]
+risk_score = 43
+rule_id = "bc1eeacf-2972-434f-b782-3a532b100d67"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
```

Comment:
doing a similar rule for Windows. this may cause some noise, seeing most noise from root as opposed to user accounts
![image](https://user-images.githubusercontent.com/56412096/106635181-85b4b880-654e-11eb-84ad-7a8c32d6e960.png)

Reply:
I think its a bit risky to exclude by user for this specific use case, there are better options (cmdline), which is often case by case.